### ISSUE TITLE
Problem loading initial RSS data

### ISSUE THREAD

**Issue by truckershitch** [GetStream/Winds#121]
I am receiving the same error on both machines when trying to load the initial RSS data in Linux with the localDiskDb connection.  I have been unsuccessful with a mysql connection as well.

Your help would be greatly appreciated.

```
[alarm@c2 Winds]$ node load_initial_data.js 
info: configuring mail service to use sendgrid backend
node_redis: Warning: Redis server does not require a password, but a password was supplied.
info: Starting to load the initial data...
info: inserted topic Design
info: inserted topic Startups
info: inserted topic Programming
info: inserted topic Gaming
info: inserted topic LifeHacks
info: inserted topic Venture Capital
info: inserted topic AI & Machine Learning
info: inserted topic News
info: inserted topic VR
info: inserted all topics, feeds are up next
info: inserted site www.designernews.co
/home/alarm/Winds/load_initial_data.js:107
                        sails.sentry.captureMessage(err)
                                    ^

TypeError: Cannot read property 'captureMessage' of undefined
    at /home/alarm/Winds/load_initial_data.js:107:37
    at wrapper (/home/alarm/Winds/node_modules/waterline/node_modules/lodash/index.js:3592:19)
    at applyInOriginalCtx (/home/alarm/Winds/node_modules/waterline/lib/waterline/utils/normalize.js:421:80)
    at wrappedCallback (/home/alarm/Winds/node_modules/waterline/lib/waterline/utils/normalize.js:335:16)
    at error (/home/alarm/Winds/node_modules/waterline/node_modules/switchback/lib/normalize.js:42:31)
    at _switch (/home/alarm/Winds/node_modules/waterline/node_modules/switchback/lib/factory.js:56:28)
    at /home/alarm/Winds/node_modules/waterline/lib/waterline/query/composite.js:71:25
    at wrapper (/home/alarm/Winds/node_modules/waterline/node_modules/lodash/index.js:3592:19)
    at applyInOriginalCtx (/home/alarm/Winds/node_modules/waterline/lib/waterline/utils/normalize.js:421:80)
    at wrappedCallback (/home/alarm/Winds/node_modules/waterline/lib/waterline/utils/normalize.js:335:16)
    at error (/home/alarm/Winds/node_modules/waterline/node_modules/switchback/lib/normalize.js:42:31)
    at _switch (/home/alarm/Winds/node_modules/waterline/node_modules/switchback/lib/factory.js:56:28)
    at /home/alarm/Winds/node_modules/waterline/lib/waterline/query/dql/create.js:73:23
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:726:13
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:52:16
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:264:21
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:44:16
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:723:17
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:167:37
    at /home/alarm/Winds/node_modules/waterline/lib/waterline/query/validate.js:79:23
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:726:13
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:52:16
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:264:21
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:44:16
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:723:17
    at /home/alarm/Winds/node_modules/waterline/node_modules/async/lib/async.js:167:37
    at _afterValidating (/home/alarm/Winds/node_modules/waterline/lib/waterline/query/validate.js:53:20)
    at allValidationsChecked (/home/alarm/Winds/node_modules/waterline/lib/waterline/core/validations.js:265:12)
```

**Comment by nparsons08** [comment]
Hi @truckershitch, can you verify that you've set all of your ENV values? Also, this application requires a MongoDB connection string, not MySQL... just a heads up.

**Comment by truckershitch** [comment]
I have set my ENV values, but I have been trying to use sails-disk and sails-mysql, as I have MySQL installed on the machine already.  I was able to get the initial data loaded by setting `migrate: alter` in `config/connections.js` and then reverting to `migrate: safe` to run the application, but I was unable to see the feeds in the application afterwards.  I have seen this working in the docker image but I was unable to successfully create an account with it.

Do I have to install MongoDB to get this to work?  I'd rather not if possible.  I had the impression that I could choose another database backend.

**Comment by nparsons08** [comment]
Unfortunately, we've pegged this version to use MongoDB. I would recommend using [MongoDB Atlas](http://mbsy.co/mongodb/228644) (they have a free offering) so you don't have to install it locally. Please let me know if that works!

**Comment by truckershitch** [comment]
Well, I put MongoDB on an Odroid C2 and I can get the initial data loaded without errors.  I can start the application, but I cannot see any feeds in the web browser, so I cannot create an account.  I don't know where the logs are so that I can show you.  Looking at the pm2 log seems to show progress, showing only errors when there is an errant feed, and I see that the data was read at your website in my dashboard. I have the mailer set to my local sendmail but I do not think this is the problem with the missing feeds.

I had used the sample .env file on this page but the trailing inline pound sign comments after the variable settings were causing problems.  I got rid of those.  I also had trouble putting in a MongoDB URI so I put the settings in manually which fixed things.

Thanks for helping me out with this.  I am stuck again so any help would be appreciated!  If you would like to see a log, I'll be happy to provide it.
